app/src/components/common/Toast.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div
+    v-if="show"
+    class="toast-container"
+    :class="styleType"
+    :style="{ animationDuration: `${timeout / 1000}s` }"
+  >
+    <font-awesome-icon
+      :icon="getIcon()"
+      style="padding-right: 10px"
```

Comment:
it is better to use one approach and describe the styles in one place -- inside the style tag for this component using css classes

Reply:
Thanks, fixed!